site/app/templates/grading/simple/Display.twig

```diff
@@ -306,9 +306,11 @@
                     data-origval="{{ component_score }}"
                     data-maxclamp="{{ max_clamp }}"
                     value="{{ component_score }}"
-                    onclick="this.select()"
+                    onclick="this.dataset.typing || this.select()"
+                    ondblclick="this.dataset.typing = true;"
+                    onblur="this.dataset.typing = false;"
                     onchange=" this.style.backgroundColor = 'var(--default-white)';"
-                    onkeydown="javascript: return ['Backspace','Delete','ArrowLeft','ArrowRight','Period'].includes(event.code) ? true : !isNaN(Number(event.key)) && event.code!=='Space'"
+                    onkeydown="return ['Backspace','Delete','ArrowLeft','ArrowRight','Period'].includes(event.code) ? true : !isNaN(Number(event.key)) && event.code!=='Space'"
```

Comment:
I think it would be good to move any nontrivial JS code here to `simple-grading.js` so everything is in one place, and is readable.  Moving JS code to a `.js` file also allows us to run our linters on the code to ensure it adheres to our code formatting standards.

Reply:
Fair enough.